### ISSUE TITLE
kdb plugin-list -v ordering

### ISSUE THREAD

**Issue by MasterToney** [ElektraInitiative/libelektra#3109]
Running the command `kdb plugin-list` returns all available plugins sorted in alphabetical order.
Running the command `kdb plugin-list -v` returns all available plugins sorted in alphabetical order trailed by the status.

Looking at [pluginlist.hpp](https://github.com/ElektraInitiative/libelektra/blob/master/src/tools/kdb/pluginlist.hpp) I can find no mention what -v should do except print verbose.

I could add a mention there that it adds info about the status of the plugins where the status is a rating on how "established" or "maintained" it is, the higher the better.

@markus2330 what would you like the behavior to be? Is it enough to mention that the status is mentioned in the help?

Personally I would argue that the output should also be sorted when using -v, what do you think?

**Comment by markus2330** [comment]
Thank you for reporting this problem!

**Comment by MasterToney** [comment]
What about the default case without -v ?
Can that stay sorted alphabetically?

We could also put the status in front (for -v) and space them the same, something like this:

```
#> kdb plugin-list -v
        7001 cache
       5600 base64 
        3450 c 
-1000000 blockresolver 
-1000000 boolean
```

That would make it obviously why they have the order they do.

**Comment by markus2330** [comment]
> What about the default case without -v ?
> Can that stay sorted alphabetically?

Yes, I found that to be an improvement, otherwise I would not have merged the PR :smile: In particular with the change `kdb plugin-list` is more friendly to beginners, which is very important for us.

> We could also put the status in front (for -v) and space them the same, something like this:

Better do not get lost in such features as they need some time to be implemented well and the benefit is marginal.

Better to fix nasty bugs such as #2380
